examples/tfengine/generated/org_foundation/devops/variables.tf

```diff
@@ -0,0 +1,59 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+variable "billing_account" {
+  type        = string
+  description = "ID of billing account to attach to this project."
+}
+
+variable "parent_id" {
+  type        = string
+  description = "ID of parent GCP resource to apply the policy. Can be one of the organization ID or folder ID according to parent_type. See https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy to learn more about resource hierarchy."
+  validation {
+    condition     = can(regex("^[0-9]{8,25}$", var.parent_id))
+    error_message = "The parent_id must be valid. Should have only numeric values with a length between 8 and 25 digits. See https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy to know how to get your organization/folder id"
+  }
+}
+
+variable "project" {
```

Comment:
Could you leave a TODO here to test with Private Catalog and make sure that can work with nested data structures like this?

Reply:
Sure. Done